.circleci/config.yml

```diff
@@ -278,7 +278,7 @@ jobs:
       - run: make init build
       - run:
           name: Status and artifacts
-          command: make depend.status
+          command: make depend.status ISTIO_OUT=/go/out
```

Comment:
Offhand I don't think ISTIO_OUT is using ?= (somewhat intentional) so I don't think it can be overriden like that.  You might have to do something more like:
<pre>make depend.status ; cp $(make where-is-out)/dep.dot /go/out/</pre>

Reply:
I tried it before I submitted :-)
```
$ make depend.status ISTIO_OUT=/tmp/go/out
dep status -dot > /tmp/go/out/dep.dot
```

Reply:
ps: I wrote makefiles with variables without ever using `?=` for about 20 years, variable setting on command line always just worked. (maybe for evil environment variables is where `=?` is needed)

Reply:
thanks for the `make where-is-out` tip; that was going to be my 2nd choice if there wasn't a variable :-)

Reply:
You're probably right.  I kinda skipped over the section about overriding variables, the precedence order, "override" directive, etc.  I suppose my excuse is being safe rather than sorry in this review.